forms-flow-web/src/components/Form/Steps/ViewAndEditTaskvariable.js

```diff
@@ -43,14 +43,16 @@ const ViewAndEditTaskvariable = ({
               setTaskLabel(e.target.value);
             }}
             className="form-control"
-            aria-labelledby="Task label"
+            aria-label="Task Label"
           />
         </td>
         <td className="p-3">
           <Form.Check
             className="" 
             disabled={enableEditTaskVariable}
             checked={showInList}
+            id="uniqueCheckboxId"
+            label="Show in List"
```

Comment:
can you add internationalisation to this label?

Reply:
goog catch @fahad-aot

Reply:
solved